Client/Frontend/Autofill/CreditCard/CreditCardBottomSheet/CreditCardBottomSheetViewModel.swift

```diff
@@ -161,25 +179,61 @@ struct CreditCardBottomSheetViewModel {
                                                                        with: ccNumberDecrypted,
                                                                        fieldValues: decryptedCreditCard)
             return updatedDecryptedCreditCard
+        case .selectSavedCard:
+            guard let row = row,
+                  let selectedCreditCard = getSavedCreditCard(for: row)
+            else {
+                return nil
+            }
+
+            let decryptedCreditCardNum = decryptCreditCardNumber(card: selectedCreditCard)
```

Comment:
⚠️ FYI: This flow uses decryption of cards which happen via rust component and initially was thinking of DI for it but that defeats the purpose for autofill. After discussing with @razvanlitianu for autofill we will add a ticket to mock it in future as @yoanarios you are already aware of tests passing locally but not in BR

Reply:
If the ticket is already created can we add a comment with the Jira link over the disabled test so we can keep track of it

Reply:
@yoanarios Added a comment here: https://github.com/mozilla-mobile/firefox-ios/pull/14986/commits/59998881080765d831d91ac90316608f347f505d